src/Weather.js

```diff
@@ -1,9 +1,6 @@
-import React, { useState, useContext } from 'react'
-import dataContext from './dataContext'
+import React from 'react'
 
-const Weather = () => {
-  const { weather, setWeather } = useContext(dataContext)
-  const [location, setlocation] = useState('test')
+const Weather = ({ weather }) => {
```

Comment:
We can do this as Weather gets a `props` object; we can destructure it "in-place" and just get the weather key.